db-schema.sql

```diff
@@ -0,0 +1,37 @@
+-- create table for different validation processes
+CREATE TABLE IF NOT EXISTS item_validation_process (
+  id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+  name VARCHAR(100) NOT NULL 
+);
+
+INSERT INTO item_validation_process (name)
+VALUES ('bad words detection'),
+  ('aggressive language and hate speech detection');
+
+-- create table for automatic validation records
+-- one record for each validation process
+CREATE TABLE IF NOT EXISTS item_validation (
+  id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+  item_id UUID NOT NULL,
+  process_id UUID NOT NULL,
+  status VARCHAR(20) NOT NULL DEFAULT 'pending',
```

Comment:
I do remember JC mentioned that we could optionally have a table for status, but I didn't see a lot of advantages for a separate table here:
There will be a lot of joins, because in several cases we want to retrieve 'pending' records from DB. And also the status is not likely to have new values / delete values.